src/Sentry.Maui.CommunityToolkit.Mvvm/MauiCommunityToolkitMvvmEventsBinder.cs

```diff
@@ -0,0 +1,159 @@
+using System.Windows.Input;
+using CommunityToolkit.Mvvm.Input;
+using Sentry.Internal;
+
+namespace Sentry.Maui.CommunityToolkit.Mvvm;
+
+/// <summary>
+/// Scans all elements for known commands that are implement
+/// </summary>
+internal class MauiCommunityToolkitMvvmEventsBinder(IHub hub) : IMauiElementEventBinder
+{
+    private const string SpanName = "ctmvvm";
+    private const string SpanOp = "relay.command";
+
+    private readonly object _propertyChangedLock = new();
```

Comment:
We probably don't need this... we can just lock against the target AsyncRelayCommand instead.